### PR TITLE
security: update x-default-browser

### PR DESCRIPTION
**Pull request by The-Code-Monkey**
Issue:

## What I did

created the package under my mono repo and published it with the security update, should work fine, i also converted all to typescript which means we don't need to declare any types 

## How to test

- [ ] Is this testable with Jest or Chromatic screenshots?
- [ ] Does this need a new example in the kitchen sink apps?
- [ ] Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

**Review by shilman — CHANGES_REQUESTED** [review]
@The-Code-Monkey looks like you need to update the import in the source code as well

**Comment by The-Code-Monkey** [comment]
@shilman bump.

**Comment by Codex-** [comment]
This addresses a new audit warning introduced in [this commit](https://github.com/storybookjs/storybook/commit/813d3bd142851ab68583b5337dfadf276b866677) so would be great to see this brought in

File: lib/core-client/src/typings.d.ts [hunk 1]
```diff
@@ -6,6 +6,5 @@ declare module 'pnp-webpack-plugin';
 declare module '@storybook/ui/paths';
 declare module 'better-opn';
 declare module 'open';
-declare module 'x-default-browser';
-declare module '@storybook/ui';
-
+declare module '@aw-web-design/x-default-browser';
+declare module '@storybook/ui';
```
Comment (Codex-) [inline lib/core-client/src/typings.d.ts:10]: removed eof newline

**Pull request merged**